docker-compose.yml

```diff
@@ -22,7 +25,16 @@ services:
       target: development
     environment:
       # Be sure to add environment variables to config/options.yml
-      - DATABASE_URL=postgres://app:secret@postgres/app_development
+      - HOSTEDGPT_DB_USERNAME=${HOSTEDGPT_DB_USERNAME:-hostedgpt_user}
+      - HOSTEDGPT_DB_PASSWORD=${HOSTEDGPT_DB_PASSWORD:-secret}
+      - HOSTEDGPT_DB_HOST=${HOSTEDGPT_DB_HOST:-postgres}
+      - HOSTEDGPT_DB_PORT=${HOSTEDGPT_DB_PORT:-5432}
+      - HOSTEDGPT_DB_NAME=${HOSTEDGPT_DB_NAME:-hostedgpt}
+      - DATABASE_URL
+      - HOSTEDGPT_DATABASE_PORT # DEPRECATED: use HOSTEDGPT_DB_PORT
+      - HOSTEDGPT_DEV_DB # DEPRECATED: use HOSTEDGPT_DB_NAME
+      - HOSTEDGPT_TEST_DB # DEPRECATED: provide HOSTEDGPT_DB_NAME and test database name is generated
+      - HOSTEDGPT_DATABASE_PASSWORD # DEPRECATED: use HOSTEDGPT_DB_PASSWORD
```

Comment:
leaving these in and marking as deprecated because if they're provided then we still want to use them to avoid breaking existing hostedgpt deployments